### PR TITLE
Change manifest to include anything under `*/templates/*`.

### PR DESCRIPTION
**Pull request by markmcd**
I'm not 100% sure what's going on, but I think the `recursive-include` statements are somehow bound to unixy paths, as they weren't being included in the built package on Windows.

Should address [this issue](https://github.com/tensorflow/tensorflow/issues/53216).

### CODE REVIEW

**Review by MarkDaoust — DISMISSED** [review]

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1 @@
-recursive-include tools/tensorflow_docs/api_generator/gen_java/ *
-recursive-include tools/tensorflow_docs/api_generator/templates/ *
\ No newline at end of file
+global-include **/templates/*
```
[the remark this answers is not in the document]
Comment (markmcd) [reply MANIFEST.in:1]: They're `gen_java/templates`, so they're covered by this.

[the remark this answers is not in the document]
Comment (markmcd) [reply MANIFEST.in:1]: Ah, I see what you mean, the shell script in gen_java wouldn't be covered by this, nor the python includes. I'll double check that now.

[the remark this answers is not in the document]
Comment (markmcd) [reply MANIFEST.in]: Fixed now. I've included *.sh across the repo & confirmed it's only added the one file.  I think that's fine, under the assumption that we want our shell scripts packaged too.

PTAL

**Review by MarkDaoust — DISMISSED** [review]

**Comment by markmcd** [comment]
Merge conflict. PTAL.

**Pull request merged**